packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -131,10 +135,11 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
             className='buy-sell__modal'
             height={table_type === buy_sell.BUY ? '400px' : '649px'}
             width='456px'
-            is_open={should_show_popup && !my_profile_store.is_cancel_add_payment_method_modal_open}
+            is_open={should_show_popup}
             title={modal_title}
             portalId={general_store.props.modal_root_id}
             toggleModal={onCancel}
+            getModalState={modalStateHandler}
```

Comment:
Looks like we are setting a value in `modalStateHandler`. 
`getModalState` doesn't properly convey the meaning. Please change it to `setModalState`

Reply:
getModalState will emit the state of the modal from the shared `modal` component. The emitted value is used by the handler function to save the value in the appropriate store associated with the custom component